aligner/u_of_michigan_aligner-checker/u_of_michigan_aligner_checker.wdl

```diff
@@ -1,26 +1,30 @@
-import "https://raw.githubusercontent.com/DataBiosphere/topmed-workflows/1.32.0/aligner/u_of_michigan_aligner/u_of_michigan_aligner.wdl" as TopMed_aligner
-import "https://raw.githubusercontent.com/DataBiosphere/topmed-workflows/1.32.0/aligner/u_of_michigan_aligner-checker/u_of_michigan_aligner_checker_calculation.wdl" as checker
+version 1.0
+
+import "https://raw.githubusercontent.com/DataBiosphere/topmed-workflows/master/aligner/u_of_michigan_aligner/u_of_michigan_aligner.wdl" as TopMed_aligner
+import "https://raw.githubusercontent.com/DataBiosphere/topmed-workflows/master/aligner/u_of_michigan_aligner-checker/u_of_michigan_aligner_checker_calculation.wdl" as checker
```

Comment:
Does using 'master' mean TravisCI will only be pulling in the latest released WDLs? I guess maybe there isn't a better way to do this until relative imports are supported in Terra so they can be used here?

Reply:
I don't remember all the details, but if I recall correctly, it has to be master because 1.32.0 is out of date and doesn't work on Terra anymore. I'd be happy to work with your team on another release.